### PR TITLE
1.2

### PR DESCRIPTION
**Pull request by Sav22999**
- Added "Recording indicator sound", if turned on it reproduce a sound when you start/stop to record a sentence (in Speak) #64 
- Added "Check for updates" option in Settings
- Added "Experimental featues" in Settings #44 
- Added shortcut in Settings: "Translate the app on Crowdin" and "See app statistics"
- Removed "Donate to the developer" because it's agains google play store policy

I've added some strings, @cloudrac3r when you have some minutes, please check them 😄

### CODE REVIEW

**Comment by cloudrac3r** [comment]
Sure thing. Can I please get a build so that I can see the context of these strings in the app?

**Comment by Sav22999** [comment]
The context is usually the same this time: Settings.
Id
If the string name is "button" ... or "text..." it is a button/option/switchOn/Off. If it is "toast..." It is a MessageDialog
________________________________

**Comment by Sav22999** [comment]
But I can provide you the apk, so you can see actual context 😄

**Comment by Sav22999** [comment]
@cloudrac3r Here you can find the apk https://send.firefox.com/download/129725d4323e9528/#JGIDaq7Bi4gXiCfEPBRdJA
(as usually, after tests, uninstall this apk please 😄 )

Thanks Sav

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -81,17 +81,28 @@
...
     <string name="txt_skip_recording_confirmation">Skip recording confirmation</string>
-    <string name="toast_skip_recording_confirmation_on">Skip recording confirmation turned on. You will now be able to send recordings without having to listen to them first.</string>
+    <string name="toast_skip_recording_confirmation_on">Skip recording confirmation turned on.\nYou will now be able to send recordings without having to listen to them first.</string>
     <string name="toast_skip_recording_confirmation_off">Skip recording confirmation turned off.</string>
+    <string name="txt_translate_the_app_on_crowdin">Translate the app on Crowdin</string>
+    <string name="txt_see_statistics_of_the_app">See official app statistics</string>
+    <string name="txt_recording_indicator_sound">Recording indicator sound</string>
+    <string name="toast_recording_indicator_sound_on">Recording indicator sound is now turned on.\nThe app will reproduce a sound when you start and stop to record a sentence.</string>
```
Comment (cloudrac3r) [inline app/src/main/res/values/strings.xml:98]: "A sound will play when you start and stop recording."

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -81,17 +81,28 @@
...
+    <string name="toast_experimental_features_on">Experimental features turned on.\nNew features could be unstable and unsafe.</string>
+    <string name="toast_experimental_featured_off">Experimental features turned off.</string>
     <string name="txt_log_on_file">Save logs to a file</string>
     <string name="toast_log_on_file_on">Saving logs to a file turned on.\nYou can attach the log.txt file when you report an issue.\nRead more on GitHub.</string>
     <string name="toast_log_on_file_off">Saving logs to a file turned off.</string>
     <string name="txt_anonymous_statistics">Anonymous statistics</string>
     <string name="toast_anonymous_statistics_on">Anonymous statistics turned on.\nWe don\'t collect your personal data. We only use statistics to see how many people use the app each day.</string>
-    <string name="toast_anonymous_statistics_off">Anonymous statistics turned off.</string>
+    <string name="toast_anonymous_statistics_off">Anonymous statistics turned off.\nWe advice to reactivate them because we don\'t collect your personal data. We only use statistics to see how many people use the app each day.</string>
```
Comment (cloudrac3r) [inline app/src/main/res/values/strings.xml:91]: "Anonymous statistics turned off. Please note that we only use statistics to see how many people use the app each day. We do not collect any personal data. Please consider turning statistics back on."

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -81,17 +81,28 @@
...
     <string name="txt_skip_recording_confirmation">Skip recording confirmation</string>
-    <string name="toast_skip_recording_confirmation_on">Skip recording confirmation turned on. You will now be able to send recordings without having to listen to them first.</string>
+    <string name="toast_skip_recording_confirmation_on">Skip recording confirmation turned on.\nYou will now be able to send recordings without having to listen to them first.</string>
     <string name="toast_skip_recording_confirmation_off">Skip recording confirmation turned off.</string>
+    <string name="txt_translate_the_app_on_crowdin">Translate the app on Crowdin</string>
+    <string name="txt_see_statistics_of_the_app">See official app statistics</string>
```
Comment (cloudrac3r) [inline app/src/main/res/values/strings.xml:96]: "official app"? What does this mean?

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Sav22999) [reply app/src/main/res/values/strings.xml:98]: Sorry, I can't understand the correct line. Could you, please, specify it for every correction? Thanks a lot

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (Sav22999) [reply app/src/main/res/values/strings.xml:96]: I don't know 😆  maybe a "copy"-"paste" wrong 🤣 
It should be "See app statistics" (without "official")

**Pull request merged**